content/nextflow-on-aws-batch/35_nextflow201/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Local Run w/ AWS Batch"
+chapter: true
+weight: 35
+---
+
+## Nextflow on AWS Batch
+
+Now that we setup AWS Batch we can use Nextflow to submit jobs we are getting closer to our architecture.
```

Comment:
Now that we setup AWS Batch, **and that** we can use Nextflow to submit jobs, we are getting closer to our **target** architecture.

*Note the comas, cannot have them in bold for whatever reasons*

Reply:
done